src/transformers/onnx/convert.py

```diff
@@ -86,6 +86,7 @@ def export_pytorch(
     opset: int,
     output: Path,
     tokenizer: "PreTrainedTokenizer" = None,
+    cuda: bool = False,
```

Comment:
Just an idea: would it be clearer to use a `device` argument here that is `cpu` by default? 

If I'm not mistaken, that allows you to have a consistent use of a `device` variable without having to hardcode `"cpu"` and `"cuda"`. Of course, you'll want a check that CUDA is available if `device="cuda"`, but this could be checked early in the script and throw a runtime error if not.

If you don't think this makes sense, my other suggestion would be to rename `cuda` to `is_cuda` to clearly indicate that it's a `bool`

Reply:
Hi @lewtun , it seems clearer to have `device` instead of `cuda`. I have just changed it in the code, thanks for the suggestion.